### ISSUE TITLE
[BUG] image transformations are not applied correctly

### ISSUE THREAD

**Issue by jiata** [microsoft/ComputerVision#336]
### Description
Bug with train/test split, looks like right now neither training nor testing gets horizontally flipped (better than both getting flipped) 

"had a look at how many transformations there are in the train and test set. Both only contained a single transformation, but the training set should be two"
